libgalois/src/analytics/connected_components/connected_components.cpp

```diff
@@ -325,9 +331,11 @@ struct ConnectedComponentsEdgeAsynchronousAlgo {
 
   using NodeData = std::tuple<NodeComponent>;
   using EdgeData = std::tuple<>;
-  typedef katana::TypedPropertyGraph<NodeData, EdgeData> Graph;
+  typedef katana::TypedPropertyGraphView<UndirectedView, NodeData, EdgeData>
+      Graph;
   typedef typename Graph::Node GNode;
-  using Edge = std::pair<GNode, typename Graph::edge_iterator>;
+  // TODO(amber): 2nd element was Graph::edge_iterator
```

Comment:
Not sure what the TODO is

Reply:
Ping

Reply:
Sorry, didn't see  your comment. 
I'm leaving it because eventually I imagine us going back to iterators instead of integer IDs.

Reply:
Ok, that makes sense